docs/guides/parallel-scraping/parallel-scraping.mdx

```diff
@@ -0,0 +1,209 @@
+---
+id: parallel-scraping-guide
+title: Parallel Scraping Guide
+sidebar_label: Parallel Scraping
+description: Parallelizing your scrapers with Crawlee
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+import SharedSource from '!!raw-loader!./shared.mjs';
+import AdaptedRoutesSource from '!!raw-loader!./adapted-routes.mjs';
+import ParallelScraperSource from '!!raw-loader!./parallel-scraper.mjs';
+import ModifiedDetailRouteSource from '!!raw-loader!./modified-detail-route.mjs';
+
+:::warning Experimental features ahead
+
+At the time of writing this guide (November 2023), request locking is still an experimental feature. You can read more about the experiment by visiting the [request locking experiment](../experiments/experiments-request-locking) page.
```

Comment:
Shouldn't this say Decemeber 2023?

Reply:
Well... when I first wrote this it was still November 🤣, but time flew and I guess a december refresh wouldn't hurt